test/pkg/environment/aws/expectations.go

```diff
@@ -31,6 +31,11 @@ func (env *Environment) ExpectInstance(nodeName string) Assertion {
 	return Expect(env.GetInstance(nodeName))
 }
 
+func (env *Environment) ExceptNodeNameToContainInstanceID(nodeName string) {
```

Comment:
since we're not using this func anywhere else, can you inline the two lines in this func in the actual test?  We can always move it to its own function later if we add more tests that use this Expectation.